internal/kafka/internal/presenters/data_plane_cluster_status.go

```diff
@@ -62,6 +71,13 @@ func getAvailableStrimziVersions(status private.DataPlaneClusterUpdateStatusRequ
 
 func PresentDataPlaneClusterConfig(config *dbapi.DataPlaneClusterConfig) private.DataplaneClusterAgentConfig {
 	accessToken := config.Observability.AccessToken
+	capacity := map[string]private.DataplaneClusterAgentConfigSpecCapacity{}
```

Comment:
is this check required?

``` d.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() ``` ?

Reply:
Not required. The check is already done in the service layer